src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f0f0f0;
+  }
+  
+  h1 {
+    color: blue;
```

Comment:
Hardcoded colors is not a good practice! Kindly, create variables to hold these values.